### ISSUE TITLE
CameraControls.zoomTo promise always resolves immediately 

### ISSUE THREAD

**Issue by looeee** [yomotsu/camera-controls#270]
I've set up a [codesandbox](https://codesandbox.io/s/camera-controls-zoomto-and-dollyto-rest-event-uxv7zf?file=/src/index.js) to test this. 

The `zoomTo` promise always resolves immediately (and `rest` event will fire immediately), while promises from other actions such as  `dollyTo` resolve once the action is complete, as expected:

Here "Zooming..." and "Finished zooming" will be logged at the same time, even though the actual zoom takes some time to complete. 

``` js
console.log("Zooming...");
await cameraControls.zoomTo(10, true);
console.log("Finished zooming.");
```

Here "Dollying..." will be logged, then the dolly will take place over a second or two, then "Finished dollying" will be logged after it's done. 

``` js
console.log("Dollying...");
await cameraControls.dollyTo(10, true);
console.log("Finished dollying.");
```

**Comment by yomotsu** [comment]
Thanks for your report as always😊
It should be fixed with #271

**Comment by looeee** [comment]
Amazing! 
You managed to get the new release out when I came back from lunch again!
Yesterday I took the afternoon off at the beach though 😁 

Checked it and it's working perfectly, thank you.
